NEWS

```diff
@@ -7,7 +7,19 @@
 
   Major changes between OpenSSL 1.1.0i and OpenSSL 1.1.1 [in pre-release]
 
-      o Support for TLSv1.3 added
+      o Support for TLSv1.3 added (see https://wiki.openssl.org/index.php/TLS1.3
+        for further important information). The TLSv1.3 implementation includes:
+          o Full support for RFC8446 (TLSv1.3) on by default
```

Comment:
I'm a little leery of the word "full", as it's a large document and I don't think that we explicitly verified compliance with all MUSTs, SHOULDs, MAYs, etc. (and I could see someone reading "full" as including all the optional stuff, too).
But I also can't think of anything off the top of my head that we didn't implement, so maybe it's fine to leave it as-is.

Reply:
We haven't done all optionals (no FFDHE springs to mind). But we should be fully compliant. Any missing MUSTs would be a bug. So perhaps reword to "Fully compliant implementation of RFC8446 (TLSv1.3) on by default"